analytics/analytics/core/job_failure_classifier/taxonomy.yaml

```diff
@@ -162,6 +162,7 @@ taxonomy:
         - 'Error: No version for .+ satisfies'
         - 'Error: errors occurred during concretization of the environment'
         - 'cannot load package .+ from the .builtin. repository'
+        - 'must have a default provider in /builds/spack/spack/etc/spack/defaults/packages.yaml'
```

Comment:
This type of error probably should be blocked by the pre-checks required before running gitlab CI.

I want to say that `spack audit packages` does this. @johnwparent do you remember?